### ISSUE TITLE
Add fastparquet package

### ISSUE THREAD

**Issue by ivandorte** [pyodide/pyodide#3170]
## 🐍 Package Request

- Package Name and Version : fastparquet / latest
- Package URL : https://github.com/MrPowers/fastparquet
- Package Dependencies that needs to be resolved first: thrift, cramjam, fsspec

**Comment by joemarshall** [comment]
Does it have an easy way to turn off threading. That was a pain when I did pyarrow build.

Does it use thrift? It isn't in the dependencies listed on GitHub? Thrift builds easily enough in emscripten anyway, so shouldn't be a problem there.

**Comment by joemarshall** [comment]
@ivandorte I built this with out of tree build tools. Fastparquet itself is easy, just `pyodide build` works there, to build cramjam (the only dependency now), I needed to install some rust stuff and fiddle to make that work. I think it shouldn't be too hard to integrate into the main package. A lot easier than pyarrow is anyway. I might have a need for it at some point soon on the job I'm doing, but I have a bunch of bits that are more on the critical path that need doing first, so I'm not sure when I'd get round to it.
